internal/pkg/grpc/client.go

```diff
@@ -122,6 +122,11 @@ func (c *Client) SendRequest(serviceMethod string, message string, headers []str
 		interpolatedHeaders[i] = placeholders.InterpolatePlaceholders(header)
 	}
 
+	if c.conn == nil {
```

Comment:
In case SendRequest() is called even after Connect() fails, it used to lead to null pointer in grpccurl.InvokeRPC()

Reply:
The error we return here is defined in a different scope, when calling `RequestParserAndFormatter`.
Can we create a new error for this scope and return it instead?